### PR TITLE
docs: Move Contribution (development) to main menu

### PR DESCRIPTION
**Pull request by dinarosv**
## Summary
To make contribution more accessible, I propose to extract Contribution to it's own page - making it reachable from the front menu. To add another Menu Card, I propose to move "Design principles" (which is only a single document) into "Design Guide", so that we continue to have 6 menu items. 

For this change, we might also need to ask some of the designers of Eufemia. ✅

## Changes
- Created Card for Contribution on front page with icon from Figma
- Moved the single page Design Principles into Design Guide
- Extracted the subpages from Contribution to actual sidebar items

### CODE REVIEW

**Review by tujoworker — APPROVED** [review]
Very good work 🙌

File: packages/dnb-design-system-portal/src/docs/contribution.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 ---
-title: 'Development'
-icon: 'development'
-order: 10
+title: 'Contribution Guide'
+description: 'Project overview, development guides, conventions etc.'
 ---
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/contribution.md:4]: We may add this redirect as well:

```mdx
redirect_from:
  - /uilib/development
```

Then links from this path will be redirected to the new place.

Comment (dinarosv) [reply packages/dnb-design-system-portal/src/docs/contribution.md:6]: Smart! Added this for `/principles` as well 😃

**Pull request merged**